webdriver/tests/bidi/network/response_started/response_started.py

```diff
@@ -172,6 +171,7 @@ async def test_response_headers(
     assert_response_event(
         events[0],
         expected_request=expected_request,
+        expected_response=expected_response,
```

Comment:
Not sure whether this should be added or if the variable assignment should be removed.

Reply:
It looks like not having this arg makes the test not verifying the response headers, which is stated in it's name.